app/src/main/java/de/danoeh/antennapod/fragment/AudioPlayerFragment.java

```diff
@@ -495,7 +501,28 @@ public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
             float prog = progress / ((float) seekBar.getMax());
             TimeSpeedConverter converter = new TimeSpeedConverter(controller.getCurrentPlaybackSpeedMultiplier());
             int position = converter.convert((int) (prog * controller.getDuration()));
-            txtvSeek.setText(Converter.getDurationStringLong(position));
+            int newChapterIndex = ChapterUtils.getCurrentChapterIndex(controller.getMedia(), position);
+            if (newChapterIndex > -1) {
+                if (!sbPosition.isPressed() && currentChapterIndex != newChapterIndex) {
+                    currentChapterIndex = newChapterIndex;
+                    position = (int) controller.getMedia().getChapters().get(currentChapterIndex).getStart();
+                    seekedToChapterStart = true;
+                    sbPosition.setIsEnabled(true);
+                    controller.seekTo(position);
+                    updateUi(controller.getMedia());
+
+                    new Handler(Looper.getMainLooper()).postDelayed(new Runnable() {
+                        @Override
+                        public void run() {
+                            sbPosition.setIsEnabled(false);
+                        }
+                    }, 5000);
```

Comment:
- When clicking multiple chapters directly after each other, the handlers could interfere. I think it would be better to move this to a `sbPosition.highlightCurrentChapter()` method that posts the runnable internally. It could then keep a fixed `Handler` instance and cancel other scheduled tasks before posting a new one.
- I find `setIsEnabled` a bit misleading because it reads like the framework method `setEnabled`. Will be fixed automatically by the comment above, though :)
- I have not tried running it on my device but I think 5 seconds might be a bit too much. How about 1 second?

Reply:
Thanks! The last commit should fix it. I had to call `invalidate()` so it is really only highlighted for a second. (And one second looks good to me :D)